src/controllers/user-controller.js

```diff
@@ -15,3 +17,17 @@ export const findAllUsers = (obj, args, { models }) => {
 export const findUserById = (_, { id }, { models }) => {
   return models.user.findByPk(id)
 }
+
+export const signIn = async (_, { data }, { models }) => {
```

Comment:
I think you're still missing the password validation.

Reply:
Hi Abel, thanks for the review. I really forgot to use standard, corrected all that. Today we had the daily with Mauricio and we agreed to do the validation of the password whenever we had any extra time, so i didn't validate still.